### ISSUE TITLE
Override method does not ovveride creating an error and failure to build for Godot 4

### ISSUE THREAD

**Issue by kilon** [Zylann/godot_voxel#445]
A new error seems to happing with lack of override 

````
[ 53%] Compiling modules/voxel/editor/instance_library/voxel_instance_library_multimesh_item_inspector_plugin.cpp ...
In file included from modules/voxel/editor/terrain/voxel_terrain_editor_plugin.cpp:1:
modules/voxel/editor/terrain/voxel_terrain_editor_plugin.h:38:37: error: 'EditorPlugin::AfterGUIInput zylann::voxel::VoxelTerrainEditorPlugin::forward_spatial_gui_input(Camera3D*, const Ref<InputEvent>&)' marked 'override', but does not override
   38 |         EditorPlugin::AfterGUIInput forward_spatial_gui_input(Camera3D *p_camera, const Ref<InputEvent> &p_event) override;
      |                                     ^~~~~~~~~~~~~~~~~~~~~~~~~

[ 53%] Compiling modules/voxel/editor/graph/voxel_graph_editor_inspector_plugin.cpp ...
In file included from modules/voxel/register_types.cpp:76:
modules/voxel/editor/terrain/voxel_terrain_editor_plugin.h:38:37: error: 'EditorPlugin::AfterGUIInput zylann::voxel::VoxelTerrainEditorPlugin::forward_spatial_gui_input(Camera3D*, const Ref<InputEvent>&)' marked 'override', but does not override
   38 |         EditorPlugin::AfterGUIInput forward_spatial_gui_input(Camera3D *p_camera, const Ref<InputEvent> &p_event) override;
      |                                     ^~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by MGilleronFJ** [comment]
I'm once again confused why I havent seen this error before. Neither did the CI... even though with the recent SCons parameters I enabled all warnings and treat them as errors...

**Comment by kilon** [comment]
> I'm once again confused why I havent seen this error before. Neither did the CI... even though with the recent SCons parameters I enabled all warnings and treat them as errors...
> 
> Which options did you use to build Godot?

the build parameters for godot apparently changed from target=release_debug to target=editor, this happened according to a commit 2 weeks ago that changes the whole way how builds are made via scons. So yes I advice to build against the latest version of Godot 4  master branch of course my build command is 
```
 scons -j20 platform=linuxbsd target=editor 
````

**Comment by Zylann** [comment]
Actually that was changed 3 hours ago... for some reason the Github blame told me it was 26 days.

**Comment by Zylann** [comment]
Should be fixed in bc9f2d6e5d944f419f77f899076d7e1e6c76c3ca

**Comment by kilon** [comment]
yeap it works fine now, thanks for the fix, issue closed
